### DISCUSSION TITLE
Tetris for Altair w/Dazzler

### DISCUSSION THREAD

**Discussion by inerlogic** [thehighnibble/thehighnibble.github.io#102] [S-132 add-on for IMSAI & Cromemco replicas]
hello, i found the following program, 

[https://github.com/phatchman/Altairtris](url)

it runs (under CP/M, no Dazzler) on my Briel Altair 8800.... i brought the .com files (tetris.com and tetrisb.com) over to the THN Imsai 8080 and of course, no luck, they appear to be running, maybe the thing wants to output to SIO2?
i don't know, i need an adultier adult to look at the source code.

i got the source to compile on the IMSAI, but it wouldn't run, i got an error: 

I>a:asm tetris
CP/M ASSEMBLER - VER 2.0
0000
000H USE FACTOR
END OF ASSEMBLY

I>a:load tetris

FIRST ADDRESS 0000
LAST  ADDRESS 0000
BYTES READ    0000
RECORDS WRITTEN 00

well, the compiler ran, but, clearly nothing came out

I>tetris

ERROR: CANNOT OPEN SOURCE, LOAD ADDRESS 0100

i need at least the console version in my life, the Dazzler and S-132 would be epic.

**Comment by nbreeden2** [comment]
I've done extensive testing on both the VT132 VT100 and Dazzler emulation working directly with the High Nibble.
On the Vt132/S-132 you need to manually change between the VT100/Dazzler screens. I believe the keyboard sequence is (press both keys at the same time):
[alt]1
[alt]5
but I could be wrong. It might be (press all three keys at the same time:
[ctrl][alt]1
[ctrl][alt]5
I don't have my system at hand to test these.
-Neil
